### DISCUSSION TITLE
How to set the first row in the data combo

### DISCUSSION THREAD

**Discussion by patriciomh1965** [serenity-is/Serenity#6180] [Help]
Hello community
I need to select by default the first row in each combo of the grid, here is the image when the option is selected in the menu:

![p1](https://user-images.githubusercontent.com/42218266/148417643-c3b7e659-b3e5-4a80-9ffc-6cbd58825d47.png)

The idea is that it appears with the initial filter at the moment the Menu option is selected.

I leave the second image as an example:
![p2](https://user-images.githubusercontent.com/42218266/148417892-2e8a7d9a-b97a-492d-a6fe-fd96d8cfcdf0.png)

Thanking you for your help
Atte.

**Comment by wezmag** [comment]
There is a sample for this. 
https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.BasicSamples/Serenity.Demo.BasicSamples/Grids/InitialValuesForQuickFilters/InitialValuesForQuickFilters.ts

**Comment by patriciomh1965** [comment]
Thank you dear, I just scheduled and it was ok. Here is the program and the image:


        protected getQuickFilters(): Serenity.QuickFilter<Serenity.Widget<any>, any>[] {

            // permite seleccionar una fila del combo
            let filters = super.getQuickFilters();

            Q.first(filters, x => x.field == fld.Transaccion).init = w => {               
                (w as Serenity.EnumEditor).value = "1"
            };
            Q.first(filters, x => x.field == fld.CncTrxConcepto).init = w => {
                (w as Serenity.EnumEditor).value = "1"
            };

            return filters;
        }
![p3](https://user-images.githubusercontent.com/42218266/148477781-4a8252ec-3d51-4189-ae87-cf48e585aa7f.png)
